gradefetcher/gradefetcher.py

```diff
@@ -304,15 +326,9 @@ def grade_user(self, data, suffix=""):
 
         reasons_msg = ""
         for reason in reasons:
-            reasons_msg += "<li>{reason}</li>".format(reason=reason)
-        self.htmlFormat = """
-        <span>
-        You got <span class="grade">{grade}% </span> score for this activity.<br />
-        Explanation: <span class="reason"><ul>{reasons_msg}</ul></span>
-        <span>
-        """.format(
-            grade=grade, reasons_msg=reasons_msg
-        )
+            reasons_msg += "<li>{reason}</li>".format(reason=reason)                
+        self.htmlFormat = self.i18n_service.gettext("You got <span class='grade'>{grade}% </span> score for this activity.<br />Explanation: <span class='reason'><ul>{reasons_msg}</ul></span>").format(
+            grade=grade, reasons_msg=reasons_msg)
```

Comment:
I wonder if it would be better to do the gettext and then apply the format to it to add the `<span...` etc.